data/human/fleets.txt

```diff
@@ -3357,3 +3357,165 @@ fleet "Human Wormhole Guards"
 		"Frigate (Mark II)"
 	variant 5
 		"Rainmaker (Mark II Old Weapons)"
+
+# The New Tortugans have access to Syndicate hulls, and any Pirate tech,
+# and more varied objectives, so their ships are a quite a mix.
+fleet "Humanika Rafeki Merchants"
+	government "Humanika Rafeki"
+	names "civilian"
+	cargo 3
+	personality
+		timid frugal opportunistic
+		confusion 20
+	variant 10
+		"Bulk Freighter (Blaster)"
+		"Aerie"
+		"Dagger"
+		"Lance"
+	variant 3
+		"Mule (Heavy)"
+		"Manta (Proton)"
+		"Dagger" 2
+	variant 3
+		"Behemoth (Heavy)"
+		"Bastion (Laser)"
+	variant 7
+		"Blackbird"
+		"Quicksilver (Proton)"
+		"Quicksilver"
+		"Raven (Afterburner)"
+	variant 7
+		"Falcon (Plasma)"
```

Comment:
Plasma turrets do not exist at the start of the game.

Reply:
I have corrected this. For some reason, I thought it had plasma cannons, not plasma turrets. Thank you for finding this for me.